### PR TITLE
CLC-4845 Filter out duplicate sections from campus data

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-4845

### CODE REVIEW

File: spec/models/campus_oracle/user_courses_spec.rb [hunk 1]
```diff
@@ -72,6 +72,13 @@
     expect(crosslisteds[0][:sections][0][:cross_listing_hash]).to eq crosslisteds[1][:sections][0][:cross_listing_hash]
   end
 
+  it 'removes duplicate sections', :if => CampusOracle::Connection.test_data? do
+    client = CampusOracle::UserCourses::All.new({user_id: '10388'})
+    courses = client.get_all_campus_courses['2013-D']
+    pp courses
```
Comment (raydavis) [inline spec/models/campus_oracle/user_courses_spec.rb]: Stray line, @paulkerschen

**Comment by raydavis** [comment]
@pfarestveit , this one looks OK for the sprint to me -- what do you think?

File: spec/models/campus_oracle/user_courses_spec.rb (hunk 1 above)
Comment (paulkerschen) [inline spec/models/campus_oracle/user_courses_spec.rb]: Ack, thanks.

**Comment by pfarestveit** [comment]
seems totally reasonable to me, @raydavis !  thanks for asking :-)

**Comment by christianv** [comment]
@paulkerschen thanks for making this change!

**Pull request merged**